### PR TITLE
PROXY-936 added purchaseContract method

### PR DESCRIPTION
**Pull request by bohdan-titan**

### CODE REVIEW

**Comment by alex-sandrk** [comment]
`package.json` => line 3, increase version to `1.0.15`

File: src/plugins/contracts/api.js [hunk 1]
```diff
@@ -202,8 +202,30 @@ function cancelContract(web3) {
...
 
+function purchaseContract(web3, cloneFactory, lumerin) {
+  return async (params) => {
+    const { walletId, contractId, url, privateKey, price } = params;
+    const sendOptions = { from: walletId, gas: 1_000_000}
+
+    const account = web3.eth.accounts.privateKeyToAccount(privateKey)
+    web3.eth.accounts.wallet.create(0).add(account)
+
+    await web3.eth.getTransactionCount(walletId, 'pending')
```
Comment (srt0422) [inline src/plugins/contracts/api.js:213]: what is this function used for?

Comment (bohdan-titan) [reply src/plugins/contracts/api.js]: Unnecessary call, that was part of what existed logic on wallet

**Comment by bohdan-titan** [comment]
Fixed

**Review by alex-sandrk — APPROVED** [review]

**Pull request merged**